exampleLitActions/src/encryptDecrypt.ts

```diff
@@ -0,0 +1,183 @@
+import { LitNodeClient } from '@lit-protocol/lit-node-client';
+import { LIT_RPC, LIT_ABILITY } from "@lit-protocol/constants";
+import {
+  createSiweMessage,
+  generateAuthSig,
+  LitActionResource,
+} from "@lit-protocol/auth-helpers";
+import * as ethers from "ethers";
+import { encryptString } from '@lit-protocol/encryption';
+
+
+// Utility function to get environment variables
+const getEnv = (name: string): string => {
+  const value = process.env[name];
+  if (!value) {
+    console.warn(`Environment variable ${name} is not set`);
+  }
+  return value || "";
+};
+
+// Environment variables
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+
+// Define the Lit Action code for conditional signing
+const litActionCode = `
+async () => {
+  const resp = await Lit.Actions.decryptAndCombine({
+    accessControlConditions,
+    ciphertext,
+    dataToEncryptHash,
+    authSig: null,
+    chain: 'ethereum',
+  });
+
+  Lit.Actions.setResponse({ response: resp });
+}
+
+`;
+
+// Define type for a basic access control condition compatible with Lit Protocol
+type LitAccessControlCondition = {
+  contractAddress: string;
+  standardContractType: string;
+  chain: string;
+  method: string;
+  parameters: string[];
+  returnValueTest: {
+    comparator: string;
+    value: string;
+  };
+};
```

Comment:
Pretty sure this is already a defined type from the Lit SDK types package, or one of our packages